site/docs/guides/create-dataflow.md

```diff
@@ -94,19 +100,28 @@ Now that you've captured data into one or more collections, you can materialize
   The page populates with the properties required for that connector.
   More details are on each connector are provided in the [connectors reference](../reference/Connectors/materialization-connectors/README.md).
 
-  Details of the collections you just created are already filled out in the **Collection Selector**.
+2. Choose a unique name for your materialization like you did when naming your capture; for example, `acmeCo/myFirstMaterialization`.
+
+3. Fill out the required properties in the **Endpoint Configuration**.
+
+4. Details of the collections you just created are already filled out in the **Collection Selector**.
+If you'd like, you can remove some of these or add additional collections.
 
-2.  Choose a unique name for your materialization like you did when naming your capture; for example, `acmeCo/myFirstMaterialization`.
+   * To easily find collections, you can use a filter.
+   Hover your cursor within to the Collection Selector table header, next to the **Remove All** button, to reveal an expandable menu icon (three dots).
```

Comment:
The menu is still shown on hover of the table header... but that is no longer next to Remove All as Remove All is in a "button row" kind of thing now.

So the three dot menu is now below the Remove All button.

![image](https://user-images.githubusercontent.com/270078/207978304-159d932a-2d70-462d-a954-8b49a9592115.png)